spec/jobs/ignored_users_summary_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+require_dependency 'jobs/scheduled/ignored_users_summary'
+
+describe Jobs::IgnoredUsersSummary do
+  before do
+    SiteSetting.ignore_user_enabled = true
+    SiteSetting.ignored_users_count_message_threshold = 1
+    SiteSetting.ignored_users_message_gap_days = 365
```

Comment:
You don't need that line if that's the default 😉

Reply:
Yes, I intentionally wrote it for other Engineers reading the specs and deciphering how this works.